doc/HOWTO-DAEMON.md

```diff
@@ -0,0 +1,159 @@
+window = <duration value>                       # default 2*cadence
+uncompleted = <bool>                            # default false
+delta-coding = <bool>                           # default true
+dump = <filename>
+```
+
+The window is the sacct time window used for looking for data.
+
+The `uncompleted` option, if true, triggers the inclusion of data about pending and running jobs.
+This will result in multiple transmissions of data for the same `(job_id,job_step)`, one at each
+sample point.  If a job stays in, say, the PENDING state for several sampling windows then multiple
+transmissions for the job in the PENDING state will be seen.
+
+The `delta-coding` option, if true, triggers optimization of data transmission: redundant data are
+omitted in subsequent transmissions, and if there are no pertient data changes - for example, if a
+new `PENDING` record has the same contents as one already sent because no settings have changed -
+then the second record will not be sent at all.  What is deemed pertient or redundant is up for
+discussion for both `PENDING` and `RUNNING` jobs.  In any case, the recipient must be prepared to
+reconstruct the complete data stream from the initial record for the `(job_id,job_step)` and any
+subsequent deltas, applied in order.  If `uncompleted` is true, the `delta-coding` option can
+greatly reduce transmitted data volume.  If `uncompleted` is false, it can still avoid redundantly
+sending information about completed jobs.
+
+Setting `dump` to a file name will cause JSON output to be appended to the given file, while also
+being sent in the normal way.  If appending fails, a diagnostic is printed if `[debug]:verbose` is
+true.
+
+### `[cluster]` section
+
+```
+cadence = <duration value>
+```
+
+### `[debug]` section
+
+```
+dump = bool                                     # default false
+verbose = bool                                  # default false
+```
+
+Setting `dump` to true will cause the daemon to dump all data it is sending on stdout.
+
+Setting `verbose` to true will cause the daemon to print somewhat informative messages about what
+it's doing at important points during the run.
+
+## DATA MESSAGE FORMATS
+
+Data messages are sent to a topic with a key and a value.
+
+Data messages are sent from Sonar to the broker under topics `<cluster>.<data-type>` where `<cluster>`
+is as configured in the `[global]` section and `<data-type>` is `sample`, `sysinfo`, `job`, `cluster`.
+
+The key sent with a message is currently the name of the originating node, including when that node
+is a master node.
+
+The values sent with these messages are opaque.  They may be a JSON object (always new-format JSON,
+see [NEW-FORMAT.md](NEW-FORMAT.md)), compressed text, and/or encrypted in some manner.  Currently
+there is no way of requesting anything other than JSON; if there is compression it is applied
+transparently, and if there is encryption it is by TLS, which must be configured explicitly in the
+config file.
```

Comment:
Reword the encryption thing since all of that was removed and will be something that comes after Kafka.